src/main/java/com/marklogic/batch/rdbms/ApplicationProperties.java

```diff
@@ -0,0 +1,72 @@
+package com.marklogic.batch.rdbms;
```

Comment:
This isn't marklogic-spring-batch code - seems like it should go in a different package, like org.example?

Reply:
I guess it depends on your overall view of this project?  I have come to look at this project as a standalone program like mlcp.  People can fork and change the code but it functions as a standalone program.  That's why I put it in a more formal package.

Reply:
Let's keep "org.example" for now. I think "com.marklogic" implies that this is MarkLogic code, but it's really intended to be part of an application separate from MarkLogic. For example, if I were to use this on a current project,  I'd immediately rename this to something project-specific. I think using "org.example" makes it more clear that this is your project's code, not ML code.